kernel/memory/src/paging/mod.rs

```diff
@@ -125,7 +129,7 @@ impl PageTable {
         other_table: &mut PageTable,
         f: F,
     ) -> Result<(), &'static str>
-        where F: FnOnce(&mut Mapper) -> Result<(), &'static str>
+        where F: FnOnce(&mut Mapper, &Mapper) -> Result<(), &'static str>
```

Comment:
update the docs for this function to describe what the second parameter to the `F` type is, the immutable `Mapper` reference.